### PR TITLE
Improve docs and add s3

### PR DESCRIPTION
**Pull request by oz123**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,7 @@ export metrics over them.
 
 ## Getting started
 
+```
```
Comment (obitech) [inline README.md]: This one actually breaks the flow

File: loki.yml [hunk 2]
```diff
@@ -39,7 +39,10 @@ schema_config:
   - from: 2018-04-15
     # Which store to use for the index.
     store: boltdb
-    object_store: filesystem
+    # this is for storing the log chunks locally
+    #object_store: filesystem
+    # uncomment bellow to store the chunks in S3
+    object_store: s3
```
Comment (obitech) [inline loki.yml]: I would rather keep this file working for local chunks + indices, and add a separate `loki-s3.yml` using S3 for chunks. Alternatively we can rename `loki.yml` into `loki-local.yml`.

File: loki.yml [hunk 3]
```diff
@@ -53,9 +56,13 @@ storage_config:
...
-  filesystem:
-    directory: ./data/loki/chunks
-
+  #filesystem:
+  #  directory: ./data/loki/chunks
+  # configures storing the chunnk in S3
+  # configure the S3 access
+  aws:
+    s3: s3://access_key:secret_access_key@rgw.noris.net/myBucket
+    s3forcepathstyle: true
```
Comment (obitech) [inline loki.yml]: See above regarding `loki-local.yml` and `loki-s3.yml`

**Review by obitech — CHANGES_REQUESTED** [review]
I'd rather not change the default `loki.yml` to use S3, but keep it for local usage. This makes it easy to launch the demo locally. 

I'd suggest creating a new `loki-s3.yml` with the necessary `storage_config` keys. We can implement a new `make` target that maybe does some `sed` to set S3 credentials and launches `docker-compose` with the right config file mounted.

File: README.md (hunk 1 above)
Comment (oz123) [reply README.md]: What do you mean? Formatting?

Comment (obitech) [reply README.md]: Yes, your added line needs to be removed again actually or the formatting is broken.

Comment (oz123) [reply README.md]: OK. Thanks for pointing this out.

File: loki.yml (hunk 2 above)
Comment (oz123) [reply loki.yml]: I split the config as you suggested.

File: loki.yml (hunk 3 above)
Comment (oz123) [reply loki.yml]: see above.

File: loki.yml (hunk 2 above)
Comment (obitech) [inline loki.yml]: This needs to be changed back to `object_store: filesystem`

File: loki.yml (hunk 3 above)
Comment (obitech) [inline loki.yml]: Needs to be changed back to:

```yaml
  filesystem:
    directory: ./data/loki/chunks
```

File: loki-s3.yml [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+      period: 168h
+
+# Configures one of many possible stores for both the index and chunks. Which
+# configuration is read from depends on the schema_config block and what is set
+# for the store value.
+storage_config:
+  # Configures storing index in BoltDB.
+  boltdb:
+    directory: ./data/loki/index
+  # Configures storing the chunks on the local filesystem.
```
Comment (obitech) [inline loki-s3.yml]: nit: remove this line

File: README.md [hunk 5]
```diff
@@ -42,6 +43,9 @@ make run/docker/up
 make run/log_gen
 ```
 
+If you have an AWS S3 bucket, you can store loki's chunks in it with
+make run/loki/s3
```
Comment (obitech) [inline README.md:51]: nit: wrap in code-block formatting and explain the user that he has to set his s3 credentials first

**Review by obitech — CHANGES_REQUESTED** [review]

File: README.md (hunk 5 above)
Comment (oz123) [reply README.md:51]: Done. Thanks for catching this.

File: loki-s3.yml (hunk 4 above)
Comment (oz123) [reply loki-s3.yml]: OK, I cleaned this.

File: loki.yml (hunk 3 above)
Comment (oz123) [reply loki.yml]: I reverted this too.

File: loki.yml (hunk 2 above)
Comment (oz123) [reply loki.yml]: reverted.

**Review by obitech — APPROVED** [review]
LGTM

**Pull request merged**